src/components/nodes-table/nodes-table.js

```diff
@@ -0,0 +1,115 @@
+      const manufacturer = node.manufacturer ? ` (${node.manufacturer})` : ''
+      return node.ready ? `${node.product}${manufacturer}` : ''
+    }
+  },
+  mounted: function () {
+    this.filters = this.initFilters()
+  },
+  computed: {
+    nodeCollection () {
+      return new NodeCollection(this.nodes)
+    },
+    relevantNodes () {
+      return this.nodeCollection
+        .filter('failed', failed => {
+          return (this.showHidden ? true : !failed)
+        })
+    },
+    filteredNodes () {
+      return this.relevantNodes
+        .betweenNumber('node_id', this.filters.ids ? this.filters.ids.min : null, this.filters.ids ? this.filters.ids.max : null)
+        .betweenDate('lastActive', this.filters.lastActives ? this.filters.lastActives.min : null, this.filters.lastActives ? this.filters.lastActives.max : null)
+
+        .contains(
+          ['product', 'manufacturer'],
+          this.filters.products ? this.filters.products.search : ''
+        )
+        .contains(['type'], this.filters.types ? this.filters.types.search : '')
+        .contains(['name'], this.filters.names ? this.filters.names.search : '')
+        .contains(['loc'], this.filters.locations ? this.filters.locations.search : '')
+        .contains(['secure'], this.filters.secures ? this.filters.secures.search : '')
+        .contains(['status'], this.filters.states ? this.filters.states.search : '')
+
+        .equalsAny('node_id', this.filters.ids ? (this.filters.ids.selections ? this.filters.ids.selections : []) : [])
+        .equalsAny('type', this.filters.types ? (this.filters.types.selections ? this.filters.types.selections : []) : [])
+        .equalsAny('product', this.filters.products ? (this.filters.products.selections ? this.filters.products.selections : []) : [])
+        .equalsAny('name', this.filters.names ? (this.filters.names.selections ? this.filters.names.selections : []) : [])
+        .equalsAny('loc', this.filters.locations ? (this.filters.locations.selections ? this.filters.locations.selections : []) : [])
+        .equalsAny('secure', this.filters.secures ? (this.filters.secures.selections ? this.filters.secures.selections : []) : [])
+        .equalsAny('status', this.filters.states ? (this.filters.states.selections ? this.filters.states.selections : []) : [])
+    },
+    tableNodes () {
+      return this.filteredNodes.nodes
+    },
+    ids () {
+      return this.relevantNodes.values('node_id')
+    },
+    products () {
+      return this.relevantNodes.values('product')
+    },
+    names () {
+      return this.relevantNodes.values('name')
+    },
+    locations () {
+      return this.relevantNodes.values('loc')
+    },
+    secures () {
+      return [undefined, false, true]
+    },
+    states () {
+      return this.relevantNodes.values('status')
+    },
+    types () {
+      return this.relevantNodes.values('type')
+    },
+    lastActives () {
+      return this.relevantNodes.values('lastActive')
+    }
+  }
+}
```

Comment:
In latest z2m version I store itemsPerPage on localstorage so when the user change the itemsPerPage it will remain the same after refreshing the page. I cannot see this here: f5ce7c2ea51337520d5512b6f944f9d35e36bcd2

Reply:
Yes, I will include that.

Reply:
@ahochsteger Did you include the commit?

Reply:
@robertsLando Not yet, but I'm working on it.

Reply:
@robertsLando It's done.